app/src/main/java/org/simple/clinic/enterotp/BruteForceOtpEntryProtection.kt

```diff
@@ -26,4 +40,69 @@ class BruteForceOtpEntryProtection @Inject constructor(
       otpState.loginAttemptFailed()
     }
   }
+
+  fun protectedStateChanges(): Observable<ProtectedState> {
+    val bruteForceProtectionResets = Observables
+        .combineLatest(config, otpStatePreference.asObservable()) { config, state -> config.blockDuration to state.limitReachedAt }
+        .switchMap { (blockDuration, blockedAt) -> signalBruteForceTimerReset(blockedAt, blockDuration) }
+        .flatMapCompletable { resetFailedAttempts() }
+        .toObservable<Any>()
+
+    return Observables
+        .combineLatest(
+            config,
```

Comment:
Same comment as above, `config` is not `Observable`. So we can directly call the params for it in `map`